COPYRIGHT

```diff
@@ -1,5 +1,6 @@
-Licensed under the Apache License, Version 2.0
-<LICENSE-APACHE or
+Licensed under the Unicode license <LICENSE-UNICODE or
```

Comment:
I would say that something like "code is licensed under ... , unicode data is licensed under ..."